website/src/components/repl/Repl.tsx

```diff
@@ -398,19 +281,31 @@ class Repl extends React.Component<Props, State> {
             <div className={styles.codePanel}>
               <Tabs
                 current={state.rightTab}
-                labels={["code"]}
+                labels={["code", "input ast"]}
```

Comment:
```suggestion
                labels={["output", "input AST"]}
```
The "AST" should be uppercase only as it is an acronym.